### PR TITLE
docs: add en-us expansion-reduction.md

### PR DESCRIPTION
**Pull request by aaronlinv**

### CODE REVIEW

**Comment by CalvinKirs** [comment]
Please take a look @Tianqi-Dotes   chinese doc is here(https://dolphinscheduler.apache.org/zh-cn/docs/latest/user_doc/expansion-reduction.html)

**Comment by dailidong** [comment]
> Please take a look @Tianqi-Dotes chinese doc is here(https://dolphinscheduler.apache.org/zh-cn/docs/latest/user_doc/expansion-reduction.html)

what’s the problem?

File: docs/en-us/1.3.5/user_doc/expansion-reduction.md [hunk 1]
```diff
@@ -0,0 +1,256 @@
...
+
+# which machine the master service is deployed on
+masters="existing master01,existing master02,ds1,ds2"
+
+# The worker service is deployed on which machine, and specify which worker group this worker belongs to, the following example of "default" is the group name
+workers="existing worker01:default,existing worker02:default,ds3:default,ds4:default"
+
+```
+
+
```
Comment (chengshiwen) [inline docs/en-us/1.3.5/user_doc/expansion-reduction.md:254]: Extra blank lines can be deleted

**Review by chengshiwen — CHANGES_REQUESTED** [review]

File: docs/en-us/1.3.5/user_doc/expansion-reduction.md (hunk 1 above)
Comment (aaronlinv) [reply docs/en-us/1.3.5/user_doc/expansion-reduction.md]: Thanks, the extra blank lines have been deleted, please check

**Review by dailidong — APPROVED** [review]
+1
good job

**Pull request merged**